libs/barista-components/breadcrumbs/src/breadcrumbs.scss

```
@@ -1,75 +1,146 @@
 @import '../../core/src/style/variables';
 @import '../../core/src/style/interactive-common';
+@import '../../core/src/style/overlay';
 @import '../../style/font-mixins';
 @import './breadcrumbs-theme';
 
 $height: 32px;
 $arrow-width: 8px;
 $arrow-padding: 2px;
 
+$dt-breadcrumb-max-width: 250px;
+$dt-breadcrumbs-panel-max-height: 256px;
+$dt-breadcrumbs-panel-max-width: 400px !default;
+$dt-breadcrumbs-panel-min-width: 112px !default;
+
 :host {
   display: flex;
-  background-color: #ffffff;
 }
 
-:host ::ng-deep a.dt-breadcrumbs-item {
+:host ::ng-deep a.dt-breadcrumbs-item,
+:host .dt-breadcrumb-collapsed-trigger {
   &,
   &:visited {
     @include dt-main-font($height);
     display: block;
     position: relative;
     height: $height;
     margin-top: 0;
-    margin-left: $arrow-padding;
-    padding: 0 10px 0 #{$arrow-width + 8px};
+    margin-left: #{-$arrow-width + $arrow-padding};
+    padding: 0 #{$arrow-width + 8px};
     color: #ffffff;
     white-space: nowrap;
     text-decoration: none;
+    z-index: 1;
+  }
+
+  &.dt-breadcrumbs-item-ellipsis {
+    text-overflow: ellipsis;
+    overflow: hidden;
   }
 
-  &:not(:last-child) {
-    @include dt-interactive-reset();
-    @include dt-cdkmonitor-focus-style();
+  &:focus {
+    outline: none;
   }
 
   &:first-child {
     margin-left: 0;
-    padding-left: 14px;
-
-    &::before {
-      display: none;
-    }
+    padding-left: 12px;
   }
 
   &:last-child {
-    flex-grow: 1;
-
     &::after {
       display: none;
     }
   }
 
   &::before,
   &::after {
-    content: ' ';
+    content: '';
     position: absolute;
     display: block;
     top: 0;
     width: $arrow-width * 2;
     height: $height;
     box-sizing: border-box;
-    border-color: transparent transparent transparent #ffffff;
+    border-color: #ffffff #ffffff #ffffff transparent;
     border-style: solid;
     border-width: #{$height / 2} $arrow-width;
+    z-index: -2;
   }
 
   &::before {
     left: 0;
-    z-index: 1;
+    width: calc(100% - #{$arrow-width});
   }
 
   &::after {
-    left: 100%;
-    z-index: 2;
+    left: calc(100% - #{$arrow-width});
+    z-index: -1;
+  }
+}
+
+.dt-breadcrumb-collapsed-trigger {
+  padding: 0 8px 0 #{$arrow-width};
+  line-height: $height;
+  @include dt-interactive-reset();
+
+  &:focus {
+    outline: none;
+  }
+}
+
+::ng-deep .dt-breadcrumbs-overlay {
+  background: #ffffff;
```

Could be the variable. 
```suggestion
  background: $white;
```